chatkit/src/main/java/com/pusher/chatkit/messages/MessageService.kt

```diff
@@ -60,9 +60,14 @@ class MessageService(
         }
     }
 
-    fun fetchMessages(): MessagesPromiseResult =
-        chatManager.doGet("/rooms/${room.id}/messages")
+    fun fetchMessages(limit: Int = -1): MessagesPromiseResult {
```

Comment:
Why `-1` as the default instead of making the parameter optional?

Reply:
True, nullable it shall be :)

Reply:
I guess the only argument against using nullable is that then we need to check for two things if it's null and higher than 0. But may be ok 🤔

Reply:
Do you have unsigned `Int`s in Kotlin?